metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/AthenaClientHelper.java

```diff
@@ -0,0 +1,125 @@
+     * Submits a query to Amazon Athena and returns the execution ID of the query.
+     * @param athenaClient
+     * @param athenaDatabase
+     * @param athenaOutputS3Bucket
+     * @param query
+     * @return Athena query execution ID
+     */
+    public static String submitAthenaQuery(AthenaClient athenaClient, String athenaDatabase, String athenaOutputS3Bucket, String query) {
+        try {
+            // The QueryExecutionContext allows us to set the database.
+            QueryExecutionContext queryExecutionContext = QueryExecutionContext.builder()
+                    .database(athenaDatabase)
+                    .build();
+
+            // The result configuration specifies where the results of the query should go.
+            ResultConfiguration resultConfiguration = ResultConfiguration.builder()
+                    .outputLocation(athenaOutputS3Bucket)
+                    .build();
+
+            StartQueryExecutionRequest startQueryExecutionRequest = StartQueryExecutionRequest.builder()
+                    .queryString(query)
+                    .queryExecutionContext(queryExecutionContext)
+                    .resultConfiguration(resultConfiguration)
+                    .build();
+
+            StartQueryExecutionResponse startQueryExecutionResponse = athenaClient.startQueryExecution(startQueryExecutionRequest);
+            return startQueryExecutionResponse.queryExecutionId();
+        } catch (AthenaException e) {
+            LOG.error("Could not submit AWS Athena query", e);
+        }
+        return "";
+    }
+
+    /**
+     * Wait for an Amazon Athena query to complete, fail or to be cancelled.
+     * @param athenaClient
+     * @param queryExecutionId
+     * @throws InterruptedException
+     */
+    public static void waitForQueryToComplete(AthenaClient athenaClient, String queryExecutionId) throws InterruptedException {
+        GetQueryExecutionRequest getQueryExecutionRequest = GetQueryExecutionRequest.builder()
+                .queryExecutionId(queryExecutionId)
+                .build();
+
+        GetQueryExecutionResponse getQueryExecutionResponse;
+        boolean isQueryStillRunning = true;
+        while (isQueryStillRunning) {
+            getQueryExecutionResponse = athenaClient.getQueryExecution(getQueryExecutionRequest);
+            String queryState = getQueryExecutionResponse.queryExecution().status().state().toString();
+            if (queryState.equals(QueryExecutionState.FAILED.toString())) {
+                throw new RuntimeException(
+                        "The Amazon Athena query failed to run with error message: " + getQueryExecutionResponse
+                                .queryExecution().status().stateChangeReason());
+            } else if (queryState.equals(QueryExecutionState.CANCELLED.toString())) {
+                throw new RuntimeException("The Amazon Athena query was cancelled.");
+            } else if (queryState.equals(QueryExecutionState.SUCCEEDED.toString())) {
```

Comment:
Or since you have a few of them, an enhanced switch on enum
https://howtodoinjava.com/java14/switch-expressions/